### ISSUE TITLE
FIN-REC-R&Q-SOA-[SOA by due date aging amount should be in 2 decimal place]

### ISSUE THREAD

**Issue by A2000-ISTEAM** [a2000-erp-team/WEBERP#765]
![image.png](https://images.zenhubusercontent.com/5e8e8265b7c1d9906c2bbc2c/907cd18a-519a-40cb-b756-18145286f5f0)

16.	Statement of Account by due date aging amount should be in 2 decimal place and consistent throughout all other column.

**Comment by jameslee888sg** [comment]
@Aungyetun991990  - for numbers/values, pls ensure it is right aligned in the cells and with 2-decimal places

@adriantoh  - pls note for subsequent tests

**Comment by Aungyetun991990** [comment]
@jameslee888sg  Well noted and  will FIx soon

**Comment by A2000-ISTEAM** [comment]
![image](https://user-images.githubusercontent.com/62376141/83221689-0b56bc00-a1a9-11ea-9a44-99cf8cbe6039.png)

Not accepted!

**Comment by Aungyetun991990** [comment]
which report still remain error?

![image](https://user-images.githubusercontent.com/47961353/83223859-fa567c80-a1a1-11ea-9b31-c6cb2af4a3ef.png)

**Comment by Aungyetun991990** [comment]
yes @jameslee888sg  I hv checked other SOA reports like By TrnDate,Doc Date.
number formats in these reports are still wrong so that i will fix the other reports soon

**Comment by A2000-ISTEAM** [comment]
Ok done.
